src/sass/app_2/_common/themes/dark.scss

```diff
@@ -736,4 +736,21 @@
             background-color: unset;
         }
     }
+    .work-in-progress {
+        background: $COLOR_DARK_BLUE_3;
```

Comment:
can you use the `@themify` mixin to update these colors in dark theme?

Reply:
sure